### PR TITLE
Decrement Member counter on Member remove

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
### Functionality:
When a member is removed from a team check if the member is in any other teams if the member is not in any of the teams then remove the member and decrement the member counter

### Solution:
changes done in removeMember.js file

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
go to Teams Details page and remove member from team

### CODE REVIEW

File: functions/model/teams/tark/removeMember.js [hunk 1]
```diff
@@ -21,23 +21,47 @@
...
 exports.removeMember = function(request, response) {
     const orgDomain = request.body.data.OrganizationDomain;
     const teamName = request.body.data.TeamName;
     const teamMembers = request.body.data.TeamMembers;
     const remove = request.body.data.Remove;
+    let flag = false;
     let result;
     let status = 200;
-
+    const res2 = {};
```
Comment (sanjaykrishna1203) [inline functions/model/teams/tark/removeMember.js:34]: This res2 can be removed

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

File: functions/model/teams/tark/removeMember.js (hunk 1 above)
Comment (Bankar-Swapnil) [reply functions/model/teams/tark/removeMember.js]: Done

**Review by simran142002 — COMMENTED** [review]
Please resolve the conflict

**Review by viv9k — APPROVED** [review]
Approved

**Review by simran142002 — COMMENTED** [review]
fix the lint issues

**Review by sanjaykrishna1203 — APPROVED** [review]
Approved

**Pull request merged**